websites/K/KBHGames/presence.ts

```diff
@@ -0,0 +1,60 @@
+const presence = new Presence({
+		clientId: "1315433043590250526",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	gameIcon = Array.from(document.styleSheets)
+		.flatMap(sheet => Array.from(sheet.cssRules || []))
+		.filter(
+			rule =>
+				rule instanceof CSSStyleRule &&
+				rule.selectorText === ".playbutton::before"
+		)
+		.map((rule: CSSStyleRule) => rule.style.backgroundImage)
+		.join("")
+		.replace(/^url\(["']?|["']?\)$/g, "");
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/c61Rzcw.png",
+		startTimestamp: browsingTimestamp,
+	};
+
+	if (window.location.hostname.includes("kbhgames")) {
```

Comment:
This check seems pointless as the Presence is only configured to run on `kbhgames.com` in the metadata.